api/src/services/import-services/utils/datetime.ts

```diff
@@ -21,15 +22,15 @@ interface ICaptureStub {
  * @returns {string | undefined}
  */
 export const formatTimeString = (time?: string | null): string | undefined => {
-  const fullTime = dayjs(time, 'HH:mm:ss');
+  const fullTime = dayjs(time, DefaultTimeFormat);
   const shortTime = dayjs(time, 'HH:mm');
```

Comment:
suggestion: This can be updated to the new `DefaultTimeFormatNoSeconds`

Reply:
Updated